src/Application/Users/Commands/CreateUserCommand/CreateUserCommand.cs

```diff
@@ -0,0 +1,51 @@
+using Application.Common.Interfaces;
+using Application.Helpers;
+using Application.Users.Queries;
+using AutoMapper;
+using Domain.Entities;
+using MediatR;
+
+namespace Application.Users.Commands.CreateUserCommand;
+
+public record CreateUserCommand : IRequest<UserDTO>
+{
+    public string Username { get; init; }
+    public string Email { get; init; }
+    public string Password { get; set; }
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public Department Department { get; set; }
```

Comment:
Who can pass the whole department object in the http request?

Reply:
ah yeah I c, understandable